source/features/softwrap-code.tsx

```diff
@@ -15,7 +15,8 @@ async function init(): Promise<void> {
 	for (const table of tables) {
 		table.classList.add('rgh-softwrapped-code');
 
-		for (const line of select.all('.blob-code-inner', table)) {
+		// `span` required to exclude `td` in PRFiles
+		for (const line of select.all('span.blob-code-inner', table)) {
```

Comment:
Adding `span` wouldn't target code inside suggestions. For suggestions that class exists on the `td` itself.